english/kitchen/prefers.md

```diff
@@ -0,0 +1,7 @@
+The chef beams, pleased that you thought to ask.
+
+[Beans](beans/beans.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.